src/main/scala/com/gu/automation/support/LocalStorageManager.scala

```diff
@@ -0,0 +1,25 @@
+package com.gu.automation.support
+
+import org.openqa.selenium.{JavascriptExecutor, WebDriver}
+
+import scala.util.Try
+
+object LocalStorageManager {
+
+  private def stripQuotes(input: String): String = {
+    return input filterNot ("'\"" contains _)
```

Comment:
Agreed, it's only to simplify debugging since people might have quotes in the value they want to store in Local Storage. From a security perspective, it's horrible, since it'd still be possible to pass in Javascript and it would execute, but for a testing framework I think it's fine  :smile: